VideoApi/VideoApi.DAL/Migrations/VideoApiDbContextModelSnapshot.cs

```diff
@@ -309,40 +309,26 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("LinkedParticipant");
                 });
 
-            modelBuilder.Entity("VideoApi.Domain.Participant", b =>
+            modelBuilder.Entity("VideoApi.Domain.ParticipantBase", b =>
                 {
                     b.Property<Guid>("Id")
                         .ValueGeneratedOnAdd()
                         .HasColumnType("uniqueidentifier");
 
-                    b.Property<string>("CaseTypeGroup")
-                        .HasColumnType("nvarchar(max)");
-
                     b.Property<Guid?>("ConferenceId")
                         .HasColumnType("uniqueidentifier");
 
-                    b.Property<string>("ContactEmail")
-                        .HasColumnType("nvarchar(max)");
-
-                    b.Property<string>("ContactTelephone")
-                        .HasColumnType("nvarchar(max)");
-
                     b.Property<long?>("CurrentConsultationRoomId")
                         .HasColumnType("bigint");
 
                     b.Property<int?>("CurrentRoom")
                         .HasColumnType("int");
 
-                    b.Property<string>("DisplayName")
-                        .HasColumnType("nvarchar(max)");
-
-                    b.Property<string>("FirstName")
-                        .HasColumnType("nvarchar(max)");
-
-                    b.Property<string>("HearingRole")
+                    b.Property<string>("Discriminator")
+                        .IsRequired()
                         .HasColumnType("nvarchar(max)");
 
-                    b.Property<string>("LastName")
+                    b.Property<string>("DisplayName")
```

Comment:
Not sure if its just github showing incorrect changes, but is this saying we have replaced the column LastName with Displayname, or does this change just reflect whats been done in the code?